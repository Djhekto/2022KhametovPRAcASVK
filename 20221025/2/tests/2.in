print(*list(slide(range(1, 20, 2), 5)))