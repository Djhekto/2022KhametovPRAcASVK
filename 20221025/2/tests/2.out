1 3 5 7 9 3 5 7 9 11 5 7 9 11 13 7 9 11 13 15 9 11 13 15 17 11 13 15 17 19 13 15 17 19 15 17 19 17 19 19